src/payment/checkout/Checkout.jsx

```diff
@@ -72,14 +72,12 @@ class Checkout extends React.Component {
     );
   };
 
-  handleSubmitStripe = () => {
-    // TODO: We'll want to submit formData here in the next iteration
-    console.log('[Project Zebra] handleSubmitStripe called');
-    this.props.submitPayment({ method: 'stripe' });
-  };
+  // handleSubmitStripe = () => {
```

Comment:
nit: is there a reason to keep this?

Reply:
It was added when I thought we were going to refactor _soon_ 😅 
I'll remove it, thanks Wes!